module/Application/config/module.config.php

```diff
@@ -4,6 +4,10 @@
  * @copyright Copyright (c) 2014 Zend Technologies USA Inc. (http://www.zend.com)
```

Comment:
Update license year please, big change in this file - removed abstract factories ! :)